artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/logger/ActiveMQAMQPProtocolMessageBundle.java

```diff
@@ -87,4 +88,26 @@
    @Message(id = 119017, value = "not authorized to create producer, {0}", format = Message.Format.MESSAGE_FORMAT)
    ActiveMQAMQPSecurityException securityErrorCreatingProducer(String message);
 
+   @Message(id = 119018, value = "link is missing a capability declaration {0}", format = Message.Format.MESSAGE_FORMAT)
+   ActiveMQBrokerConnectionException missingCapability(String capability);
```

Comment:
```suggestion
   @Message(id = 119018, value = "link is missing an offered capability declaration {0}", format = Message.Format.MESSAGE_FORMAT)
   ActiveMQBrokerConnectionException missingOfferedCapability(String capability);
```

I would also suggest it would be nicer if this was next to/consecutive with the related 'missingDesiredCapability' message (currently id 119024).